client/blocks/reader-avatar/index.jsx

```diff
@@ -32,7 +43,7 @@ const ReaderAvatar = ( { author, siteIcon, feedIcon, siteUrl, siteIconSize = 96,
 		};
 	}
 
-	let hasSiteIcon = !! ( fakeSite && fakeSite.icon );
+	let hasSiteIcon = !! ( fakeSite && fakeSite.icon.img );
```

Comment:
Yep, this was a bug I spotted along the way!

We could just do `!! get( fakeSite, 'icon.img' );`?